### PR TITLE
update MMPD

### PR DESCRIPTION
**Pull request by McJackTang**

### CODE REVIEW

**Comment by xliucs** [comment]
@McJackTang  Looks fine to me. Can you pelase provide your baseline results (MAE, MAPE etc.) at this PR?

**Comment by McJackTang** [comment]
In the simplest scenerio（include the stationary, skin tone type 3, and artificial light conditions）of MMPD，TSCAN model trained on PURE got MAE: 1.46, RMSE: 3.33, MAPE: 1.98, PEARSON: 0.94 as benchmark.

**Review by xliucs — APPROVED** [review]
LGTM

File: main.py [hunk 1]
```diff
@@ -183,6 +184,11 @@ def unsupervised_method_inference(config, data_loader):
             raise ValueError("Unsupported dataset! Currently supporting UBFC, PURE, and SCAMPS.")
 
         if config.TRAIN.DATA.DATASET is not None and config.TRAIN.DATA.DATA_PATH:
+=======
```
Comment (xliucs) [inline main.py:187]: @McJackTang  Are you sure your main file could work? Please test it before commit it.

**Pull request merged**